controller.py

```diff
@@ -74,3 +80,24 @@ def update_url(time=last_update_time, page=None):
             )
 
         return feed_response(update_feed)
+
+    def remove_items(self):
```

Comment:
I don't know how big a deal this is, but it seems weird that you might successfully remove some items from your collection and then get an error message because of invalid URNs. 

In the `lookup` handler I resolve this dilemma (you gave a bunch of URNs, some of them worked, some didn't) by serving an OPDS feed where every entry has a `message`. The message might be 'success!' or it might be 'invalid URN'. How much work would it be to do things that way here?